### PR TITLE
added some Data Access folks, made some fixes

### PR DESCRIPTION
**Pull request by jbecla**

### CODE REVIEW

File: O3-1.tex [hunk 1]
```diff
@@ -22,37 +22,40 @@
 
 \title{The LSST Data Management System}
 \author{
-Mario~Juric,$^1$
-Jeffrey~Kantor,$^2$
```
Comment (timj) [inline O3-1.tex:26]: These commas were in the right place. The example latex file distributed by the conference committee has the commas before the superscript.

Comment (jbecla) [inline O3-1.tex:26]: ok. (looks weird though, I think :). That is why I did it in a separate commit

Comment (timj) [inline O3-1.tex:26]: Well, the separate commit means I can't just hit the merge button and have to do a manual cherry pick :smile:

Comment (jbecla) [inline O3-1.tex:26]: Better than copying and pasting my changes ;).
I can redo the branch and force push, would you like me to?

Comment (timj) [inline O3-1.tex:26]: I'd prefer the force push with the single commit. Otherwise I'll cherry pick (I obviously want you to be credited with the change).

Comment (jbecla) [inline O3-1.tex:26]: done

**Pull request merged**